frontend/src/app/mock-queries.ts

```diff
@@ -1,21 +1,75 @@
 import { Query } from './query';
```

Comment:
I have many optional points to raise on this kind of file:
 - please document them (file comment at least). Mocks and tests are the best example of how the programmer designed its code.
 - maybe you should move this to another directory, since it is not part of the code (e.g. frontend/src/app/{tests,mock}, depending your use case)